nautobot/extras/tests/integration/__init__.py

```diff
@@ -0,0 +1,32 @@
+import uuid
+
+from nautobot.extras.models import Status
+from nautobot.dcim.models import Device, DeviceRole, DeviceType, Manufacturer, Site
+
+
+def _create_test_device():
```

Comment:
I don't think this needs to be named as a private function, especially as you're calling it from other files. `create_test_device` should be fine.